converter/converter.go

```diff
@@ -0,0 +1,437 @@
+package converter
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/scanner"
+
+	"github.com/emicklei/proto"
+	"github.com/knq/snaker"
+)
+
+// pkgOpt stores a package option and its comments.
+// ie: "go_package" or "java_package".
+type pkgOpt struct {
+	typeName string // go_package or java_package, etc.
+	value    string
+	comment  *proto.Comment
+}
+
+type builder struct {
+	// The converted proto to gunk declarations. This only stores
+	// the messages, enums and services. These get converted to as
+	// they are found.
+	translatedDeclarations []*strings.Builder
```

Comment:
Seems oddly specific to store `*strings.Builder` here. Why not `string` or `fmt.Stringer`?

Reply:
(same applies to the types passed around by functions)